content/conditions/constipation/main-content.md

```diff
@@ -0,0 +1,161 @@
+
+Simple changes to your child’s diet and potty training can help treat constipation.
+
+You may notice a difference within a few days. Sometimes it takes a few weeks before their symptoms improve.
+
+### Make changes to your child’s diet
+
+Give your baby extra water between their normal feeds if they haven’t started to eat solid food yet. If you’re using formula milk, don’t add more water to the mixture.
+
+Try gently moving your baby's legs in a bicycling motion or carefully massaging their tummy to help stimulate their bowels.
+
+Give older children plenty of fluids and encourage them to eat fruit. Chop or puree it if it’s easier for them to eat. The best fruits for constipation include apples, grapes, pears and strawberries.
+
+Don’t force your child to eat as this can make mealtimes stressful.
+
+[What to feed young children](http://www.nhs.uk/Conditions/pregnancy-and-baby/Pages/understanding-food-groups.aspx).
+
+### Helping your child with potty training
+
+Some children feel anxious or stressed about using the toilet. This can cause them to hold in their poo and lead to constipation.
+
+This usually happens during potty training or if their usual toilet routine has changed. For example, after moving house or starting nursery.
+
+Give your child plenty of time to use the toilet while they are still learning. Encourage them when they do use the toilet. Some parents find a reward chart works.
+
+[Potty training tips](http://www.nhs.uk/conditions/pregnancy-and-baby/pages/potty-training-tips.aspx)
+
+
+## Your pharmacist can help with constipation
+
+Speak to your pharmacist if diet and lifestyle changes aren’t helping. They can suggest a suitable laxative. These are medicines that help you poo more regularly.
+
+Most laxatives work within 3 days. They should only be used for a short time.
+
+Laxatives are not recommended for children unless they are prescribed by your GP.
+
+[Different types of laxatives and how they work](https://www.nhs.uk/conditions/laxatives/Pages/Introduction.aspx).
+
+!!! info
+  [Find your nearest pharmacy]({{ findersBaseUrl }}finders/find-help)
+!!!
+
+!!! attention
+  ## See a GP if you or your child:
+  
+  - are not improving with treatment
+  - are regularly constipated and it lasts a long time 
+  - have [blood in your poo](http://www.nhs.uk/conditions/rectal-bleeding/Pages/Introduction.aspx)
+  - have unexpectedly lost weight
+  - feel very tired all the time
+  
+  Speak to your doctor before you stop taking any prescribed medication.
+!!!
+
+!!!info
+  If you’re not sure what to do, call 111.
+!!!
+
+
+## Complications of long-term constipation
+
+Long-term constipation can lead to faecal impaction. This is where poo has built up in your rectum.
+The main symptom is [diarrhoea](http://www.nhs.uk/conditions/diarrhoea/pages/introduction.aspx) after a long bout of constipation.
+
+Faecal impaction may be treated with: 
+
+- stronger laxatives - prescribed by your GP
```

Comment:
Feels like this should be an emdash rather than a dash? Same for other items in this list